docs/src/guides/integration/create-identity.md

```diff
@@ -1,59 +1,35 @@
-
-1. In MetaMask, a user signs a message with the private key of their Ethereum account.
-2. In your dapp, the user creates a deterministic identity \[link] with the signed message.
-3. The user can now recreate their Semaphore identity whenever they want by signing the same message with their Ethereum account in Metamask.
-
-Semaphore identities can also be created using other data (such as web2 reputations from [Interep](https://interep.link/)) or using random numbers.
-
-[@src/background/zk-keeper.ts](https://github.com/privacy-scaling-explorations/crypt-keeper-extension/blob/master/src/background/zk-keeper.ts)
-
-```typescript
-this.add(
-            RPCAction.CREATE_IDENTITY,
-            LockService.ensure,
-            async (payload: NewIdentityRequest) => {
-                try {
-                    const { strategy, messageSignature, options } = payload
-                    if (!strategy) throw new Error('strategy not provided')
-
-                    const numOfIdentites = await this.identityService.getNumOfIdentites()
-                    const config: any = {
-                        ...options,
-                        name: options?.name || `Account # ${numOfIdentites}`
-                    }
-
-                    if (strategy === 'interrep') {
-                        console.log("CREATE_IDENTITY: 1")
-                        config.messageSignature = messageSignature;
-                        console.log("CREATE_IDENTITY: 2")
-                    }
-
-                    const identity: ZkIdentityWrapper | undefined = await identityFactory(strategy, config)
-                    console.log("CREATE_IDENTITY: 4", identity);
+By default, CryptKeeper uses either its supported internal ethereum wallet, or an external wallet like MetaMask (and still the only supported external wallet) to create a new Semaphore Identity.
+
+1. In CryptKeeper/MetaMask wallet, a user signs a message with the private key of their Ethereum account.
+2. In your DApp, the user creates a deterministic identity with the signed message.
+3. The user can now recreate their Semaphore identity whenever they want by signing the same message with their Ethereum account in CryptKeeper/MetaMask wallet.
+
+&#x20;[@src/providers/sdk/CryptKeeperInjectedProvider.ts](https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/29b211bae48573f0212ce280fdcc2c4a5932394b/packages/providers/src/sdk/CryptKeeperInjectedProvider.ts#L197-L210)
+
+```ts
+  /**
+   * Connects to an existing identity for the specified host.
+   *
+   * @param {IConnectIdentityRequestArgs} host - The host for which to connect to an identity.
+   * @returns {Promise<void>} A Promise that resolves when the connection is complete.
+   */
+  async connectIdentity({ host }: IConnectIdentityRequestArgs): Promise<void> {
+    await this.post({
+      method: RPCAction.CONNECT_IDENTITY_REQUEST,
+      payload: {
+        host,
+      },
+    });
+  }
+```
```

Comment:
The same here. Why do we need this code? How will it help to use provider methods?